File: gadle-dependency/java-demo/src/test/java/com/chen/demo/GadleDependencyApplicationTests.java
package com.chen.demo;

import org.junit.Test;

public class GadleDependencyApplicationTests {

    @Test
    public void contextLoads() {
        JavaUser javaUser = new JavaUser();
        javaUser.setId(2);

        assert (javaUser.getId() == 2);
    }

}

File: gadle-dependency/kotlin-demo/src/test/java/com/chen/demo/GadleDependencyApplicationTests.kt
package com.chen.demo

import org.junit.Test

//@RunWith(SpringRunner::class)
//@SpringBootTest
class GadleDependencyApplicationTests {

    @Test
    fun contextLoads() {
        val javaUser = JavaUser()
        javaUser.id = 2

        assert(javaUser.id == 2)
        print("kotlin-test success")
    }

}
